### PR TITLE
Updates min_enclosing_triangle.cpp

### PR DESCRIPTION
**Pull request by sturkmen72**
### This pullrequest changes

a small improvement on minEnclosingTriangle

### CODE REVIEW

File: samples/cpp/minarea.cpp [hunk 1]
```diff
@@ -8,13 +8,10 @@ using namespace std;
...
-         << "of points using functions: minAreaRect() minEnclosingTriangle() minEnclosingCircle().\n"
-         << "Random points are generated and then enclosed.\n\n"
-         << "Press ESC, 'q' or 'Q' to exit and any other key to regenerate the set of points.\n\n"
-         << "Call:\n"
-         << "./minarea\n"
-         << "Using OpenCV v" << CV_VERSION << "\n" << endl;
+	cout << "This program demonstrates finding the minimum enclosing box, triangle or circle of a set\n"
+		<< "of points using functions: minAreaRect() minEnclosingTriangle() minEnclosingCircle().\n"
+		<< "Random points are generated and then enclosed.\n\n"
+		<< "Press ESC, 'q' or 'Q' to exit and any other key to regenerate the set of points.\n\n";
```
Comment (alalek) [inline samples/cpp/minarea.cpp]: Please use 4 spaces instead of tabs.

**Comment by vpisarev** [comment]
@sturkmen72, when you round the triangle vertex coordinates, the new triangle is not necessarily enclosing nor minimal. Is this extension really important? I believe, if user really wants integer vertices and understands the implications of rounding, he/she can do the conversion in a single line of code. I suggest to close this PR.

**Comment by sturkmen72** [comment]
@vpisarev you are right. let me undo changes on minEnclosingTriangle. other parts are related on documentation. let me keep them.

**Comment by vpisarev** [comment]
@sturkmen72, ok, thank you! :+1:

**Pull request merged**